x-pack/plugins/apm/server/lib/helpers/aggregated_transactions/get_fallback_to_transactions.ts

```diff
@@ -8,6 +8,10 @@
 import { getSearchAggregatedTransactions } from '.';
 import { SearchAggregatedTransactionSetting } from '../../../../common/aggregated_transactions';
 import { Setup, SetupTimeRange } from '../setup_request';
+import { kqlQuery, rangeQuery } from '../../../../../observability/server';
+import { ProcessorEvent } from '../../../../common/processor_event';
+import { TRANSACTION_DURATION } from '../../../../common/elasticsearch_fieldnames';
+import { APMEventClient } from '../create_es_client/create_apm_event_client';
 
 export async function getFallbackToTransactions({
```

Comment:
Nit: can we make it clear this returns a boolean

```suggestion
export async function getDidFallbackToTransactions({
```

or 

```suggestion
export async function getWillFallbackToTransactions({
```

or

```suggestion
export async function getIsUsingTransactionEvents({
```